bin/asciibinder

```diff
@@ -104,6 +115,23 @@ Description:
   Creates a new, bare AsciiBinder repo in the specified directory.
 EOF
     end
+  when "clone"
+    Trollop::options do
+      banner <<-EOF
+Usage:
+  #$0 clone <options> <git_repo_url>
+
+Description:
+  Clones an existing AsciiBinder repo to the current directory.
+  Under the default behavior, AsciiBinder will attempt to set up
+  tracking branches based on the contents of _distro_map.yml,
+  but this can be suppressed (see Options).
+
+Options:
+EOF
+      opt :nobranches, "Don't attempt to create tracking branches after cloning.", :type => :bool, :default => false
```

Comment:
With trollop, avoid starting an option with "no", because the library has built in capabilities for boolean logic.

Prefer something like:

```ruby
opt :branches, "Create tracking branches after cloning.", :default => true
```

This allows the user to say `--branches false`, but also creates an option allowing the user to do `--no-branches` which inverts the logic automatically.

Also, no need to specify the type if you specify a default...it's automatically inferred.